content/rs/administering/database-operations/migrate-to-active-active.md

```diff
@@ -0,0 +1,96 @@
+---
+Title: Migrate a Database to Active-Active
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+With [Active-Active databases]({{< relref "/rs/administering/designing-production/active-active.md" >}}), applications can read and write to the same data set from different geographical locations seamlessly and with latency less than 1 ms, without changing the way the application connects to the database.
+Active-Active databases also provide disaster recovery and accelerated data read-access for geographically distributed users.
+
+If you have data in a single-region RS database that you want to migrate to an Active-Active database,
+you'll need to create a new Active-Active database and migrate the data into the new database as a [Replica Of](https://docs.redislabs.com/latest/rs/administering/active-passive/) the existing database.
+This process will gradually populate the data in the Active-Active database.
+
+Before data migration starts, all data is flushed from the Active-Active database.
+The data is migrated to the Active-Active instance where you enabled Replica Of, and then the Active-Active replication copies the data from that instance to the other Active-Active instances.
```

Comment:
```suggestion
The data is first migrated to the Active-Active instance where you enabled Replica Of, and then the data from that instance is copied to the other Active-Active instances.
```

Reply:
@kaitlynmichaelredislabs It's actually an ongoing process. The "first step" doesn't need to get completed before the sync starts. Very quick they will happen simultaneously

Reply:
Reworded to remove the order of events.